packages/react-native-sdk/docusaurus/docs/reactnative/06-advanced/06-deeplinking.mdx

```diff
@@ -1,194 +1,193 @@
+#### Creating Intent Filters
 
-    - `action` - Specify the ACTION_VIEW intent action so that the intent filter can be reached from Google Search.
+To create a link to your app content, add an intent filter that contains these elements and attribute values in your `AndroidManifest.xml` file at `/android/app/src/main/AndroidManifest.xml`.
 
-    ```xml
-    <action android:name="android.intent.action.VIEW" />
-    ```
+- `action` - Specify the ACTION_VIEW intent action so that the intent filter can be reached from Google Search.
 
-    - `data` - Add one or more `<data>` tags, each of which represents a URI format that resolves to the activity. At minimum, the `<data>` tag must include the `android:scheme` attribute.
+```xml
+<action android:name="android.intent.action.VIEW" />
+```
 
-    ```xml
-    <data android:scheme="http" />
-    <data android:scheme="https" />
-    <data android:host="`YOUR URL HERE`" />
-    ```
+- `data` - Add one or more `<data>` tags, each of which represents a URI format that resolves to the activity. At minimum, the `<data>` tag must include the `android:scheme` attribute.
+
+```xml
+<data android:scheme="http" />
+<data android:scheme="https" />
+<!-- The URL here must exclude the scheme -->
+<data android:host="`YOUR URL HERE`" />
+```
+
+- `category` - Include the `DEFAULT` and the `BROWSABLE` category. It is required in order for the intent filter to be accessible from a web browser. Without it, clicking a link in a browser cannot resolve to your app.
 
-    - `category` - Include the `DEFAULT` and the `BROWSABLE` category. It is required in order for the intent filter to be accessible from a web browser. Without it, clicking a link in a browser cannot resolve to your app.
+```xml
+<category android:name="android.intent.category.DEFAULT" />
+<category android:name="android.intent.category.BROWSABLE" />
+```
 
-    ```xml
+Combined together your intent filter should look like:
+
+```xml
+<intent-filter android:autoVerify="true">
+    <action android:name="android.intent.action.VIEW" />
     <category android:name="android.intent.category.DEFAULT" />
     <category android:name="android.intent.category.BROWSABLE" />
-    ```
-
-    Combined together your intent filter should look like:
+    <data android:scheme="http" />
+    <data android:scheme="https" />
+    <!-- Example: "stream-calls-dogfood.vercel.app” -->
+    <data android:host="`YOUR URL HERE`" />
+</intent-filter>
+```
 
-    ```xml
-    <intent-filter android:autoVerify="true">
-      <action android:name="android.intent.action.VIEW" />
-      <category android:name="android.intent.category.DEFAULT" />
-      <category android:name="android.intent.category.BROWSABLE" />
-      <data android:scheme="http" />
-      <data android:scheme="https" />
-      <!-- Example: "https://stream-calls-dogfood.vercel.app” -->
-      <data android:host="`YOUR URL HERE`" />
-    </intent-filter>
-    ```
+#### Verifying the link
```

Comment:
```suggestion
#### Adding Asset Links
```
Keeps the naming similar to the other section `Adding Associated Domains Entitlement`